tox.ini

```diff
@@ -15,8 +15,7 @@ deps =
     -r tests/requirements.txt
     cov: codecov
 commands =
-    bash ./tests/test.sh
-    nocov: python -m unittest discover -s tests/jobwatcher -p "*tests.py"
+    nocov: python -m unittest discover -p "*tests.py"
```

Comment:
we don't need this cause the only unit test which was not in pytest has been removed

Reply:
Removed, thanks